ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -456,7 +456,9 @@ def yield_data_models(self, dbt_objects: DbtObjects) -> Iterable[DataModelLink]:
                             self.metadata,
                             entity_type=Table,
                             service_name=self.config.serviceName,
-                            database_name=manifest_node.database,
+                            database_name=None
+                            if manifest_node.database == "None"
```

Comment:
can this only happen for the database? Also it might be better to have a controlled list of elements that we consider as "missing". `None` can be one, not sure if users reported similar strings like `null`, `none`, etc. In any case, we can have a separated constant to track this out.

Reply:
makes sense will add a constant list and include these keywords

Reply:
@pmbrull have made the changes can you please take a look now